pkg/configuration/config_manager/config_handler.go

```diff
@@ -288,6 +377,28 @@ func createConfigVolumeMeta(configSpecName string, reloadType appsv1alpha1.CfgRe
 	}
 }
 
+func isBatchReload(configMeta *ConfigSpecInfo) bool {
+	isBatchReload := false
+	if configMeta != nil &&
+		configMeta.ReloadOptions != nil &&
+		configMeta.ReloadOptions.ShellTrigger != nil &&
+		configMeta.ReloadOptions.ShellTrigger.BatchReload != nil {
+		isBatchReload = *(configMeta.ShellTrigger.BatchReload)
+	}
+	return isBatchReload
+}
```

Comment:
Is this better?

```
return configMeta != nil &&
       configMeta.ReloadOptions != nil &&
       configMeta.ReloadOptions.ShellTrigger != nil &&
       configMeta.ReloadOptions.ShellTrigger.BatchReload != nil &&
       *onfigMeta.ReloadOptions.ShellTrigger.BatchReload
```

Reply:
Fixed in d5abe20ac8a0254c609bdd3ff573125f5fa3b3b1